src/wasm/rlottiewasm.cpp

```diff
@@ -105,5 +256,26 @@ EMSCRIPTEN_BINDINGS(rlottie_bindings)
         .constructor(&RlottieWasm::create)
         .function("load", &RlottieWasm::load, allow_raw_pointers())
         .function("frames", &RlottieWasm::frames)
-        .function("render", &RlottieWasm::render);
-}
+        .function("render", &RlottieWasm::render)
+        .function("set_fill_color", &RlottieWasm::setFillColor)
```

Comment:
use CamelCase function ex: setFillColor  instead of ste_fill_color . and the rest of the Apis.

Reply:
Thank you!, I will make you new pull request! :D